mobile/src/components/chat-conversation/components/messages-list/components/date-separator/date-separator.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from 'react';
+
+import { Text, View } from '~/components/common/common';
+import { checkIsToday, getFormattedDate } from '~/helpers/helpers';
+
+import { styles } from './styles';
+
+type Props = {
+  messageTime: string;
+};
+
+const DateSeparator: FC<Props> = ({ messageTime }) => {
+  const isToday = checkIsToday(new Date(messageTime));
+  const date = isToday ? 'Today' : getFormattedDate(messageTime, 'dd MMM');
```

Comment:
Add also check for current year as it's done in this PR https://github.com/BinaryStudioAcademy/bsa-2022-guruhub/pull/522